zerver/tests/test_signup.py

```diff
@@ -365,7 +367,7 @@ def get_reset_mail_body(self, subdomain: str = "zulip") -> str:
         self.assertEqual(self.email_envelope_from(message), settings.NOREPLY_EMAIL_ADDRESS)
         self.assertRegex(
             self.email_display_from(message),
-            rf"^Zulip Account Security <{self.TOKENIZED_NOREPLY_REGEX}>\Z",
+            rf'^{_("Zulip Account Security")} <{self.TOKENIZED_NOREPLY_REGEX}>\Z',
```

Comment:
What's the story behind this change? I'd expect this translation to be a noop...

Reply:
This is added to ensure that the language code change in `zerver/form.py` gets properly tested. The test case `test_wrong_subdomain_i18n` overrides the currently active language.